### PR TITLE
Add endpoint for EVSS 

### PR DESCRIPTION
**Pull request by annaswims**
## Description of change
For form526 BDD we need to add separation location, we can get that data from the EVSS reference data endpoint intakesites.  

Previously, there were two other endpoints that were calling EVSS reference data endpoints through an old/unused authentication scheme so I stripped that out. That means we no longer need the sockisify gem.

## Original issue
department-of-veterans-affairs/va.gov-team#9735

### CODE REVIEW

File: app/controllers/v0/apidocs_controller.rb [hunk 1]
```diff
@@ -173,7 +173,6 @@ class ApidocsController < ApplicationController
       Swagger::Schemas::ConnectedApplications,
       Swagger::Schemas::Dependents,
       Swagger::Schemas::DismissedStatus,
-      Swagger::Schemas::Form526::RatedDisabilities,
```
Comment (annaswims) [inline app/controllers/v0/apidocs_controller.rb:176]: duplicate

File: config/settings.yml [hunk 2]
```diff
@@ -194,9 +195,6 @@ evss:
     cert_path: ~
     key_path: ~
     root_ca: ~
-  jwt:
```
Comment (annaswims) [inline config/settings.yml:197]: unused

File: config/settings.yml [hunk 3]
```diff
@@ -444,10 +442,6 @@ betamocks:
   cache_dir: /cache
   services_config: config/betamocks/services_config.yml
 
-faraday_socks_proxy:
```
Comment (annaswims) [inline config/settings.yml:447]: unused - TODO: check if I need to remove from devops repo.

File: spec/request/address_request_spec.rb [hunk 4]
```diff
@@ -166,18 +167,5 @@
         end
       end
     end
-
-    context 'with a 401 malformed token response', vcr: { cassette_name: 'evss/reference_data/401_malformed' } do
```
Comment (erluti) [inline spec/request/address_request_spec.rb:170]: how come this test/cassette was removed?

Comment (annaswims) [reply spec/request/address_request_spec.rb:170]: There used to be header/JWT code that was unique to this controller, now it uses the same header that the rest of the EVSS calls use.

**Comment by annaswims** [comment]
this pr will fix the EVSS connection in review instances https://github.com/department-of-veterans-affairs/devops/pull/7107

File: spec/request/swagger_spec.rb [hunk 5]
```diff
@@ -517,6 +517,13 @@
         end
       end
 
+      it 'supports getting separation_locations' do
+        expect(subject).to validate(:get, '/v0/disability_compensation_form/separation_locations', 401)
```
Comment (annaswims) [inline spec/request/swagger_spec.rb:521]: Reconsider this test

Comment (annaswims) [inline spec/request/swagger_spec.rb:521]: @SarahJaine What I missed earlier when we were talking was the `headers` argument.  The 401 response is unauthenticated.

**Pull request merged**